services/datamanager/builtin/builtin.go

```diff
@@ -426,12 +428,16 @@ func (svc *builtIn) Reconfigure(
 	}
 	svc.collectors = newCollectors
 
+	if svc.syncDisabled == svcConfig.ScheduledSyncDisabled &&
+		svc.syncIntervalMins == svcConfig.SyncIntervalMins &&
+		reflect.DeepEqual(svc.additionalSyncPaths, svcConfig.AdditionalSyncPaths) {
```

Comment:
Only use reflection when necessary because it's incredibly slow, should use a string slice method here